retroarch-standalone-service/retroarch-standalone.xinitrc

```diff
@@ -6,10 +6,18 @@ xset -dpms         # disable DPMS (Energy Star) features.
 xset s noblank     # don't blank the video device
 
 # Hide mouse cursor
-unclutter -display :0.0 -noevents -grab -root -reset &
+unclutter -display :0.0 -root &
 
 # Clean old debug logs
 rm /home/retroarch/debug.log 2>&1 >&/dev/null
 
 # Autostart Retroarch
-exec /usr/bin/retroarch --appendconfig=/etc/retroarch-standalone.cfg -v 2>&1 >&/home/retroarch/debug.log
+exit_code=1
+while [ "$exit_code" -ne "0" ]
+do
+  /usr/bin/retroarch --appendconfig=/etc/retroarch-standalone.cfg -v --log-file /home/retroarch/debug.log
+  exit_code=$?
+done
+
+sudo shutdown -h now
```

Comment:
Does this work out of the box when we install the package? Or does it require some more work on the user end? Maybe we can add a check if:
1) `sudo` exists in the platform and
2) if the user has sudo rights

If both are true, than a shutdown could be issued.

Thank you in advance :)

Reply:
I'm thinking a better approach here is to try to run another script in the retroarch user's directory if the file is present. Something like
```
if [ -x "/home/retroarch/.xinit_exit" ]
then
  /home/retroarch/.xinit_exit
fi
```

Reply:
I like the idea, works better and you can customize its behavior easily :)